pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
 
-- Show process(es) that using socket:
-
-`ss -p`
-
 - Filter TCP sockets by states, only/exclude:
 
 `ss {{state/exclude}} {{bucket/big/connected/synchronized/...}}`
 
-- Filter sockets by address and/or port:
+- Show all TCP sockets connected to the local HTTPS port:
+
+`ss -t src :{{443}}`
+
+- Show all TCP sockets along with processes connected to a remote ssh port:
+
+`ss -pt dst :{{ssh}}`
+
+- Show all UDP sockets connected on a local port within a range:
 
-`ss -t dst 1.2.3.4:80`
-`ss -u src 127/8`
-`ss -t 'dport >= :1024'`
-`ss -x "src /tmp/.X11-unix/*"`
-`ss -t state established '( dport = :ssh or sport = :ssh )'`
+`ss -u 'sport >= :{{lowest_port}} and sport <= :{{highest_port}}'`
```

Comment:
I'll change this example to use both sport and dport,  should be very intuitive that way